src/Illuminate/Console/Enums/ScheduledMonth.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Illuminate\Console\Enums;
+
+enum ScheduledMonth: string
```

Comment:
```suggestion
enum ScheduledMonth: int
```

Reply:
Damn, good catch 👍🏻 Thought, I changed it 🤦🏻